app/src/main/java/com/sdex/activityrunner/manifest/ManifestViewerActivity.kt

```diff
@@ -7,26 +7,29 @@ import android.graphics.Color
 import android.os.Bundle
 import android.view.Menu
 import android.view.MenuItem
+import android.view.View
 import android.widget.Toast
+import androidx.activity.OnBackPressedCallback
 import androidx.activity.result.contract.ActivityResultContracts
 import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatDelegate
-import androidx.appcompat.widget.SearchView
+import androidx.appcompat.widget.Toolbar
+import androidx.core.view.isVisible
 import com.sdex.activityrunner.BuildConfig
 import com.sdex.activityrunner.R
 import com.sdex.activityrunner.commons.BaseActivity
 import com.sdex.activityrunner.databinding.ActivityManifestViewerBinding
 import com.sdex.activityrunner.db.cache.ApplicationModel
 import com.sdex.activityrunner.preferences.AppPreferences
 import com.sdex.activityrunner.util.IntentUtils
-import com.sdex.activityrunner.util.UIUtils
 import com.sdex.activityrunner.util.highlightjs.models.Language
 import com.sdex.activityrunner.util.highlightjs.models.Theme
+import com.yupo.browserfiplib.FiPSearchView
 import dagger.hilt.android.AndroidEntryPoint
 import javax.inject.Inject
 
 @AndroidEntryPoint
-class ManifestViewerActivity : BaseActivity() {
+open class ManifestViewerActivity : BaseActivity() {
```

Comment:
Why do you need this class to be open?

Reply:
Sorry, Android Studio auto-changed code, will fix it.